accounts/urls.py

```diff
@@ -1,5 +1,13 @@
-from django.conf.urls import url, include
 from django.urls import path
-from django.contrib import admin
+from rest_framework.urlpatterns import format_suffix_patterns
+from . import views
+
+app_name = 'accounts'
+urlpatterns = [
+    path('', views.index, name='index'),
+    path('listAPI/', views.AccountListAPI.as_view(), name='account_api'),
```

Comment:
`API` feels kinda redundant, `list` should be enough

Reply:
okay